Lib9c.sln

```diff
@@ -1,5 +1,6 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
+# 
```

Comment:
Is this mark needed?

Reply:
I have no idea as I didn't add that.  It apparently was added by `dotnet sln add` command.

Reply:
🆗